examples/plotting/add_countour_with_spherical_screen.py

```diff
@@ -0,0 +1,57 @@
+"""
+=============================================
+Add contours to a map using a SphericalScreen
+=============================================
+
+In this example, we try to show that SphericalScreen can be used as a general method
+to transform the coordinates(change the frame of reference i.e. the observer)
+from one image to another as seen by a new observer, even without reprojection.
+Here, we will draw contours on one image and then use the `SphericalScreen`
+class to draw the contours on another image using the former's perspective.
+"""
+
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+
+import sunpy.data.sample
+import sunpy.map
+from sunpy.coordinates.screens import SphericalScreen
+
+##############################################################################
+# For this example, we will start with the sample data. To create the maps,
+# we need the 171 and 193 AIA images.
+
+aia193_map = sunpy.map.Map(sunpy.data.sample.AIA_193_IMAGE)
+aia171_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+##############################################################################
+# Create a figure with two subplots. The first subplot will have the 171 image
+# with contours drawn on it. The second subplot will have the 193 image drawn
+# on it.
+fig= plt.figure(figsize=(15, 5))
+ax1 = fig.add_subplot(1, 3, 1, projection=aia171_map)
+aia171_map.plot(axes=ax1, clip_interval=(1,99.9)*u.percent)
+aia171_map.draw_contours(levels=[1,5]*u.percent, axes=ax1, colors='C0')
+ax1.set_title("Contours on 173")
+ax2 = fig.add_subplot(1,3,2,projection=aia193_map)
+aia193_map.plot(axes=ax2)
```

Comment:
```suggestion

fig = plt.figure()
ax1 = fig.add_subplot(1, 3, 1, projection=aia171_map)
aia171_map.plot(axes=ax1, clip_interval=(1,99.9)*u.percent)
aia171_map.draw_contours(levels=[1, 5]*u.percent, axes=ax1, colors='C0')
ax1.set_title("Contours on 173")
ax2 = fig.add_subplot(1, 3, 2, projection=aia193_map)
aia193_map.plot(axes=ax2)
```
So you plan to show the before and after contours?

Reply:
without the `figsize` the deafault way the images are shown are a bit overlapping
![image](https://github.com/user-attachments/assets/5bec014c-89c6-4a5b-8122-d51af34aa0cf)


so you will have to resize the image.

Reply:
We should fix some of the labels and maybe try a tight layout and then if we still need to, change the figure size.